src/functions/multiples/multiples.js

```diff
@@ -0,0 +1,15 @@
+// JavaScript Document
+function solution(number) {
```

Comment:
please try to make the function name meaningful
The name should tell what the function is doing